sqlglot/expressions.py

```diff
@@ -6006,6 +6006,10 @@ class CurrentUser(Func):
     arg_types = {"this": False}
 
 
+class UtcDate(Func):
+    arg_types = {}
```

Comment:
We should investigate if any of the supported dialects supports `UTC_DATE`. Introducing a new `Func` subclass can result in parser errors if its `arg_types` is incompatible with a signature for that function.

For example... if BigQuery supported `UTC_DATE(arg)`, it would break because:

1. The parser automatically recognizes `UTC_DATE` and tries to build a `UtcDate` from it
2. The `arg_types` defined here specifies that no arguments are expected, so validation fails

Reply:
MySQL and Oracle have the `UTC_DATE` function with the same signature
w3resource.com/mysql/date-and-time-functions/mysql-utc_date-function.php
https://docs.oracle.com/cd/E17952_01/mysql-5.7-en/date-and-time-functions.html#function_utc-date

Reply:
I checked other dialects and didn't find `UTC_DATE` function

Reply:
Thank you